network-api/networkapi/wagtailpages/factory/homepage_features.py

```diff
@@ -18,8 +18,6 @@ def generate(seed):
 
     reseed(seed)
 
-    # These are "guaranteed" to exist as they are created as
-    # part of the wagtailpages migrations:
```

Comment:
Hm, this  comment is correct, though: the wagtailpages migrations include creating the new Areas of Focus objects, so rather than removing this comment, really it's the _code_ that needs updating. (e.g. `get_or_create`)

Reply:
let's revert this change, seeing how it's not publications-related.

Reply:
Ok, will revert!